b2b.mk

```diff
@@ -58,3 +58,15 @@ b2b.watch_css:
 b2b.fix-npm:
 	docker-compose `echo ${DOCKER_COMPOSE_FILES}` exec b2b bash -c 'chown -R root ~/.npm'
 
+b2b.createsuperuser:
+	docker-compose `echo ${DOCKER_COMPOSE_FILES}` exec b2b python manage.py createsuperuser --settings=edraakprograms.dev
+
+b2b.css:
+	docker-compose `echo ${DOCKER_COMPOSE_FILES}` exec b2b npm run scss
+
+b2b.makemessages:
```

Comment:
@qutaiba I'm not sure we need to `makemessages` on setup. since the po files are checked in, we just need to compile them to get the compiles mo version. This way every time the setup is run, the po files will appear as changes to be committed.

Reply:
@a-nassif `makemessages` is needed on your local setup to collect the new added strings, and po files are required to be committed as mo files are ignored and will not be committed, `compilemessages` will happen during the deployment